sycl/test-e2e/ESIMD/unified_memory_api/Inputs/atomic_update_slm.hpp

```diff
@@ -116,14 +113,14 @@ bool test_slm(queue q) {
     auto e = q.submit([&](handler &cgh) {
       cgh.parallel_for(rng, [=](id<1> ii) SYCL_ESIMD_KERNEL {
         int i = ii;
-        slm_init<32768>();
+        constexpr uint32_t SLMSize = N * (stride + 1) * sizeof(T);
```

Comment:
Why does SLMSize need a different formula?
Looking at how SLM is initialized (slm_block_store<T, size>(0, data);) and read then to copy SLM to arr for verification, this line should be simpler and also unified with init/read-verify code:
```suggestion
        constexpr uint32_t SLMSize = size * sizeof(T);
```

Reply:
Done, thanks!